src/kernel/capinv.c

```diff
@@ -39,12 +39,25 @@ fs_reg_setup(unsigned long seg) {
 		      : : "b" (seg));
 }
 
+static void
+tlb_mandatory_flush(void *arg)
```

Comment:
I'm not sold on this in the kernel API.  Is this just a stop-gap measure for the publication?  Shouldn't we replace this with async threads later?  This is OK for now, but put a `TODO` comment above it and create a github issue.